redash/destinations/webex.py

```diff
@@ -0,0 +1,158 @@
+        description = alert.custom_body if alert.custom_body else ""
+
+        if new_state == "triggered":
+            if alert.custom_subject:
+                subject = alert.custom_subject
+            else:
+                subject = alert.name + " just triggered"
+        else:
+            subject = alert.name + " went back to normal"
+
+        attachments = [
+            {
+                "contentType": "application/vnd.microsoft.card.adaptive",
+                "content": {
+                    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+                    "type": "AdaptiveCard",
+                    "version": "1.0",
+                    "body": [
+                        {
+                            "type": "ColumnSet",
+                            "columns": [
+                                {
+                                    "type": "Column",
+                                    "width": 4,
+                                    "items": [
+                                        {
+                                            "type": "TextBlock",
+                                            "text": subject,
+                                            "weight": "bolder",
+                                            "size": "medium",
+                                            "wrap": True,
+                                        },
+                                        {
+                                            "type": "TextBlock",
+                                            "text": description,
+                                            "isSubtle": True,
+                                            "wrap": True,
+                                        },
+                                        {
+                                            "type": "TextBlock",
+                                            "text": f"Click [here]({query_link}) to check your query!",
+                                            "wrap": True,
+                                            "isSubtle": True,
+                                        },
+                                        {
+                                            "type": "TextBlock",
+                                            "text": f"Click [here]({alert_link}) to check your alert!",
+                                            "wrap": True,
+                                            "isSubtle": True,
+                                        },
+                                    ],
+                                },
+                                {
+                                    "type": "Column",
+                                    "width": 1,
+                                    "items": [
+                                        {
+                                            "type": "Image",
+                                            "url": "https://cdn.pixabay.com/photo/2016/09/16/09/20/alarm-1673577_960_720.png",
```

Comment:
I don't think we should use this CDN's image directly like this (even if it seems royalty-free). They don't require credits as per the [ToS](https://pixabay.com/service/terms/) but we should probably credit them anyway if we want to use their image.